aws-frauddetector-detector/src/aws_frauddetector_detector/handler_workers.py

```diff
@@ -41,10 +41,6 @@ def execute_create_detector_handler_work(session: SessionProxy, model: models.Re
     if model.Arn is not None or model.CreatedTime is not None or model.LastUpdatedTime is not None:
         raise exceptions.InvalidRequest("Error occurred: cannot create read-only properties.")
 
-    # API does not handle 'None' property gracefully
-    if model.Tags is None:
-        del model.Tags
-
```

Comment:
We handle none gracefully now in the update_tags function. This was causing a bug when models did not contain Tags.